### PR TITLE
Communication Guidelines edit

### PR DESCRIPTION
**Pull request by aaronepperson**
This updates the communication guidelines with edits and other updates for style guide formats.

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by mrbobbytables** [comment]
/assign 
to get it on my to-do list

**Comment by parispittman** [comment]
thanks for the PR! I've only done a quick skim and I'm finding a lot of empty [ ]'s after [link descriptions]. examples in the moderation docs. will do a formal review in a bit.

File: communication/mailing-list-guidelines.md [hunk 1]
```diff
@@ -1,293 +1,270 @@
...
-(like spamming) should be moderated.
-- [Lock the thread immediately] so that people cannot reply to the thread.
-- [Delete the post].
-- In some cases you might need to ban a user from the group, follow
-  [these instructions] on how stop a member from being able to post to the 
-  group.
-  For more technical help on how to use Google Groups, check the [Groups Help]
-  page.
+When a user violates the [Code of Conduct][] or performs other negative
+activities (like spamming or harassment), moderators should:
```
Comment (jberkus) [inline communication/mailing-list-guidelines.md:44]: we should also add a bullet about contacting the CoCC here.

**Comment by mrbobbytables** [comment]
@parispittman 
> thanks for the PR! I've only done a quick skim and I'm finding a lot of empty [ ]'s after [link descriptions]. examples in the moderation docs. will do a formal review in a bit.

The empty [] is another way of representing a reference link and is often put in there by markdown formatters.

Markdown is great for being flexible, but also annoying at how inconsistent it is lol

**Comment by parispittman** [comment]
@mrbobbytables do we have one way or another listed in the style guide? or are we accepting of all ways for markdown?

File: communication/discuss-guidelines.md [hunk 2]
```diff
@@ -1,188 +1,161 @@
...
 
 If the problem is with one of the Admins or Moderators, reach out to one of the
 other Admins and describe the situation.
 
-If it is a [Code of Conduct] issue, contact conduct@kubernetes.io and describe
-the situation.
-
----
+If it is a [Code of Conduct][Kubernetes Code of Conduct] issue, contact
+conduct@kubernetes.io and describe the situation.
```
Comment (jberkus) [inline communication/discuss-guidelines.md:71]:
```suggestion
our [Code of Conduct Committee](mailto: conduct@kubernetes.io) and describe the situation.
```

**Review by aaronepperson — COMMENTED** [review]
Accepted changes that added COCC bullet point

**Review by markyjackson-taulia — APPROVED** [review]

**Comment by mrbobbytables** [comment]
@parispittman we don't have both ways listed in the style guide, but I'd personally accept both - I've seen both used with the trailing[] frequently injected by linters / auto formatters.

File: communication/moderation.md [hunk 3]
```diff
@@ -5,79 +5,59 @@ description: |
...
-  - [Discuss](#discuss)
-  - [Mailing List](#mailing-list)
-  - [Slack](#slack)
-  - [YouTube](#youtube)
-  - [Zoom](#zoom)
-  - [Process for Adding a Moderator](#process-for-adding-a-moderator)
-- [References and Resources](#references-and-resources)
-
----
+-   Check the [centralized list of administrators][] for contact information.
```
Comment (mrbobbytables) [inline communication/moderation.md:13]:
```suggestion
-   Check the [centralized list of administrators][moderators] for contact information.
```

File: communication/moderation.md [hunk 4]
```diff
@@ -5,79 +5,59 @@ description: |
...
+-   Some Kubernetes properties, like the Twitter account, are managed by the
+    [Cloud Native Computing Foundation][].
 
 ## Selection of Moderators
 
-Each Kubernetes property has a certain set of [moderators] who
-are responsible for keeping it safe and a fun place to participate.
+Each Kubernetes property has a certain set of
+[moderators][centralized list of administrators] who are responsible for keeping
```
Comment (mrbobbytables) [inline communication/moderation.md:20]:
```suggestion
[moderators][] who are responsible for keeping
```

File: communication/moderation.md [hunk 5]
```diff
@@ -112,88 +90,90 @@ the Kubernetes community, and it is your responsibility to remain aware of your
...
-- Help your colleagues if you recognize them in one of the [stages of burnout].
-- Be helpful and have fun!
-
+Moderators *MUST*:
+
+-   Take action as specified by these Kubernetes Moderator Guidelines.
+    -   You are empowered to take *immediate action* when there is a violation.
+        You do not need to wait for review or approval if an egregious
+        violation has occurred. Make a judgment call based on our [Code of
+        Conduct][] and Values (see below).
```
Comment (mrbobbytables) [inline communication/moderation.md:99]:
```suggestion
        violation has occurred. Make a judgment call based on our [Code of Conduct][] and Values (see below).
```

links that span multiple lines break Hugo rendering :(

File: communication/moderation.md [hunk 6]
```diff
@@ -202,58 +182,56 @@ umbrella.
...
+    OFF in the settings.
+-   [Youtube Guidelines][]
+-   [Moderators][17]
 
 ### Zoom
 
-- [Zoom Guidelines](./zoom-guidelines.md)
-- [Moderators](./moderators.md#zoom)
+-   [Zoom Guidelines][]
+-   [Moderators][18]
```
Comment (mrbobbytables) [inline communication/moderation.md:212]: This entire block should be updated removing the numbers from the links and be given a friendly name or link in the list directly.

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Thanks! left some suggestions to markdown changes :)

File: communication/mailing-list-guidelines.md [hunk 7]
```diff
@@ -1,293 +1,270 @@
...
 
+-   At least 3 mailing list owners (leads), in addition to
+    contributors@kubernetes.io
+
+-   Familiarity with the [moderation guidelines][] for the project and
+    [moderation queues][moderation queue]. Chairs should be cognizant that a
+    new group will require an initial time investment moderation-wise as the
+    group establishes itself.
 
 ### Create the leads and members mailing lists
```
Comment (jberkus) [inline communication/mailing-list-guidelines.md:96]: @mrbobbytables should we be instructing people to create lists under kubernetes.io now?

Note:  the instructions below this are just reformatting of the instructions already there, so it shouldn't hold up the PR approval.  However, it raises a question for future revisions.

**Review by jberkus — CHANGES_REQUESTED** [review]
Approved if you  make the pending changes.

**Comment by jberkus** [comment]
Please make the cited changes, particularly Bob's fixes to the links.  Then can merge.  Sorry for the long review period, there's a lot of lines in this one!

File: communication/mailing-list-guidelines.md (hunk 7 above)
Comment (mrbobbytables) [reply communication/mailing-list-guidelines.md:96]: I'd do that in a separate commit or separate PR from the style edits, I wouldn't necessarily block on it.

Comment (jberkus) [reply communication/mailing-list-guidelines.md:96]: Right.  More of a flag as a TODO.  Lemme open an issue.

File: communication/mailing-list-guidelines.md (hunk 1 above)
Comment (aaronepperson) [reply communication/mailing-list-guidelines.md:44]: Hey! @jberkus can I confirm what you mean by the CoCC? I'm not sure what that is referring to. Thanks in advance!

**Comment by pymander** [comment]
@jberkus @mrbobbytables I worked with Aaron to do tricky Git stuff and resolve all of the issues we could do, then I did a rebase and force push.

**Comment by jberkus** [comment]
/approved

@parispittman @mrbobbytables can one of you LGTM it?

**Comment by jberkus** [comment]
Grrr.

/approve

File: communication/mailing-list-guidelines.md (hunk 1 above)
Comment (jberkus) [reply communication/mailing-list-guidelines.md:44]: Code of Conduct Committee

Comment (pymander) [reply communication/mailing-list-guidelines.md:44]: @jberkus I addressed this in the latest push!

File: communication/moderation.md [hunk 8]
```diff
@@ -262,19 +239,40 @@ allowing us to build on the shoulders of giants. Moderators are encouraged to
...
+  [zoom-mods]: ./moderators.md#zoom
+  [filing an issue]: https://github.com/kubernetes/community/issues/new/?assignees=&labels=area%2Fcommunity-management%2C+sig%2Fcontributor-experience&template=moderator_application.md&title=REQUEST%3A+New+moderator+for+<your-GH-handle>+of+<k8s+property>
+  [kubernetes/k8s.io]: https://github.com/kubernetes/k8s.io
+  [moderators@kubernetes.io group]: https://github.com/kubernetes/k8s.io/blob/main/groups/groups.yaml
+  [Forum Moderation Guidelines]: https://support.mozilla.org/en-US/kb/moderation-guidelines
+  [How to Moderate a Mailing List]: https://www.oasis-open.org/khelp/kmlm/user_help/html/mailing_list_moderation.html
+  [How to effectively moderate forums]: http://www.communityspark.com/how-to-effectively-moderate-forums/
+  [5 tips for more effective community moderation]: https://www.socialmediatoday.com/social-business/5-tips-more-effective-community-moderation
+  [8 Helpful Moderation Tips for Community Managers]: https://sproutsocial.com/insights/tips-community-managers/
+  [Setting Up Community Guidelines for Moderation]: https://www.getopensocial.com/blog/community-management/setting-community-guidelines-moderation
```
Comment (mrbobbytables) [inline communication/moderation.md:278]: definitely has to be a linter, do you know why its doing that?

**Review by mrbobbytables — CHANGES_REQUESTED** [review]
Thanks!
Just have some suggested link fixes and a few questions regarding the linter/formatter^^;;

File: communication/discuss-guidelines.md [hunk 9]
```diff
@@ -191,31 +152,31 @@ below.
...
+  [Slack Channel]: https://slack.k8s.io/
+  [built in system for flagging inappropriate posts]: https://meta.discourse.org/t/what-are-flags-and-how-do-they-work/32783
+  [unlist]: https://meta.discourse.org/t/what-is-the-difference-between-closed-unlisted-and-archived-topics/51238
+  [user trust system]: https://blog.discourse.org/2018/06/understanding-discourse-trust-levels/
+  [moderation guidelines]: ./moderation.md
+  [Site Feedback and Help]: https://discuss.kubernetes.io/c/site-feedback
+  [SIG, WG or subjproject]: /sig-list.md
+  [Regional Discussions Category]: https://discuss.kubernetes.io/c/regional-discussions
+  [translated versions of the CNCF Code of Conduct]: https://github.com/cncf/foundation/tree/master/code-of-conduct-languages
+  [CNCF foundation]: https://github.com/cncf/foundation
```
[the remark this answers is not in the document]
Comment (pymander) [reply communication/discuss-guidelines.md:182]: @mrbobbytables The auto-formatter adds all of those lines. I can certainly remove them!

[the remark this answers is not in the document]
Comment (jberkus) [reply communication/discuss-guidelines.md:182]: hey, is there any chance you can reformat this?  Otherwise, I'll close it.

**Pull request closed without merging**